packages/contracts-bedrock/test/L2/SuperchainTokenBridge.t.sol

```diff
@@ -60,6 +61,31 @@ contract SuperchainTokenBridgeTest is Bridge_Initializer {
         superchainTokenBridge.sendERC20(address(superchainERC20), ZERO_ADDRESS, _amount, _chainId);
     }
 
+    /// @notice Tests the `sendERC20` function reverts when the `token` does not support the IERC7802 interface.
+    function testFuzz_sendERC20_notSupportedIERC7802_reverts(
```

Comment:
I propose to add a 
```
vm.mockCall(
    _token, abi.encodeCall(ISuperchainERC20.supportsInterface, (type(IERC7802).interfaceId)), abi.encode(true)
);
 ```
on the `setUp` that will work for each test except this where it will be overwritten.
This is because, I remember one reason of some flakky test we got was that it fuzzed over a token that implemented the function but wasn't compatible with the interface. With the initial mock to true, you can enforce that.

Reply:
hmm the setup is using a real `SuperchainERC20` being deployed by the factory, so the integration case is the most similar to a real case, i dont think we should mock this function

Reply:
ah true, is not fuzzing the token address. Nvm then 👍